src/Money.php

```diff
@@ -171,6 +171,18 @@ public function rawFormatted(mixed ...$overrides): string
         ]));
     }
 
+    /** Create a Money instance from a formatted string. */
+    public static function fromFormatted(string $formatted, Currency|string $currency = null, mixed ...$overrides): self
```

Comment:
Probably add a full method docblock explaining what the parameters are for. Most importantly, that the `$overrides` means the overrides used for formatting the original Money instance into the passed string.

Also, why is `$overrides` a variadic array here? Does that mean `fromFormatted('$ 10.15', 'USD', prefix: '$ ')` is supported?

Reply:
> Probably add a full method docblock explaining what the parameters are for.

We already have type hints for the method. Adding docblocks is redundant.

![image](https://user-images.githubusercontent.com/11887260/160977928-712f5d5a-fa50-4a11-843d-67514ef89ac9.png)

---

> Most importantly, that the $overrides means the overrides used for formatting the original Money instance into the passed string.

How about improving the readme instead?

![image](https://user-images.githubusercontent.com/11887260/160977978-57561858-18c1-4f73-bf02-0aba39311c0f.png)

---

> Also, why is $overrides a variadic array here? Does that mean fromFormatted('$ 10.15', 'USD', prefix: '$ ') is supported?

Yes, same as the `formatted()` method.

Reply:
> We already have type hints for the method. Adding docblocks is redundant.

I meant adding descriptions to parameters, not types.

And improving the readme is fine, I'd just make the method understandable on its own as well.

Reply:
Done